src/servicelevelagreement/models/clause.cto

```diff
@@ -11,6 +11,8 @@ asset ServiceLevelAgreementContract extends AccordContract {
  o Double serviceCredit1
```

Comment:
If you change this type to be `MonetaryAmount`, it means that the logic doesn't need to hard-code the currency code

Reply:
Hi Matt, I changed the variables from Double to Monetary Amount, but the codehas not picked up the changes I made and are still expecting a Double value. Is this something you have seen before. [ergo] This operator received an unexpected argument of type `Double'